mcs/class/aot-compiler/Makefile

```diff
@@ -38,29 +38,34 @@ LIBRARY_INSTALL_DIR = $(mono_libdir)/mono/$(FRAMEWORK_VERSION)
 
 ifndef SKIP_AOT
 
+profile_file=$(wildcard $(topdir)/class/lib/build/csc.aotprofile)
+ifneq ($(profile_file),)
+profile_arg=,profile=$(profile_file)
```

Comment:
AOT is already enabled, this just adds a few instances to it, not much right now, since the main roslyn assemblies are not AOTed.